CHANGELOG.md

```diff
@@ -6,11 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [v1.0.0] - 2020-02-07
+## [v1.2.0] - 2020-05-19
```

Comment:
Looks like your branch is out of date.  There should also be a v1.1.0 in the changelog.  Can you rebase?  

You'll need to add a new entry for this release instead of modifying an existing entry.  You'll also need to add a matching link at the bottom of the file and update the link for Unreleased:
```
[Unreleased]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.2.0...HEAD

[v1.2.0]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.1.0...v1.2.0
[v1.1.0]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.0.0...v1.1.0
[v1.0.0]: https://github.com/CruGlobal/cru-ansible-modules/releases/tag/v1.0.0
```

Reply:
Looks like the releases got mixed up. They should be descending order, with unreleased on the top:

```
## [Unreleased]

## [v1.2.0] - 2020-05-22
<list v1.2.0 changes here>

## [v1.1.0] - 2020-04-22
### Added
- New module `splitout`
### Updated
- Use Python 3 for local action modules
- Various fixes

## [v1.0.0] - 2020-02-07
### Added
- This CHANGELOG file
- Basic CONTRIBUTING template

[Unreleased]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.2.0...HEAD

[v1.2.0]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.1.0...v1.2.0
[v1.1.0]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.0.0...v1.1.0
[v1.0.0]: https://github.com/CruGlobal/cru-ansible-modules/releases/tag/v1.0.0
```